06-05 16:26:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-172]
06-05 16:29:23 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "cosObjId" not found [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3047)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3133)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at models.User.retrieveUser(User.java:320)
	at controllers.Application.getToken(Application.java:88)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:369)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:369)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
06-05 16:30:08 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "cosObjId" not found [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3047)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3133)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at models.User.retrieveUser(User.java:320)
	at controllers.Application.getToken(Application.java:88)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:369)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:369)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
06-05 16:30:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1413)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1388)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:428)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:255)
	at com.jolbox.bonecp.ConnectionHandle.recreateConnectionHandle(ConnectionHandle.java:281)
	at com.jolbox.bonecp.ConnectionHandle.close(ConnectionHandle.java:512)
	at models.User.retrieveUser(User.java:347)
	at controllers.Application.getToken(Application.java:88)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:369)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:369)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
06-05 17:41:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Column "cosObjId" not found [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3047)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3133)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at models.User.retrieveUserCOS(User.java:263)
	at controllers.Application.getToken(Application.java:89)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:373)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:373)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
06-05 17:41:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "cosObjId" not found [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3047)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3133)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at models.User.retrieveUserCOS(User.java:263)
	at controllers.Application.getToken(Application.java:89)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:373)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:373)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
06-05 17:42:38 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1413)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:967)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1574)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:941)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:99)
	at com.jolbox.bonecp.PreparedStatementHandle.executeQuery(PreparedStatementHandle.java:174)
	at models.User.retrieveUserCOS(User.java:260)
	at controllers.Application.getToken(Application.java:89)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:373)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$17$$anonfun$apply$17.apply(routes_routing.scala:373)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
06-05 17:58:58 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "cosObjId" not found [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3047)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3133)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at models.User.retrieveUser(User.java:318)
	at controllers.Application.getCosObj(Application.java:105)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$18$$anonfun$apply$18.apply(routes_routing.scala:381)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$18$$anonfun$apply$18.apply(routes_routing.scala:381)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
